superset/views/core.py

```diff
@@ -3132,15 +3182,18 @@ def _load_data_from_columns(self, table_name, columns):
         :param columns: The names of columns to select
         :return: The data from columns from given table as list of tuples
         """
-        selected_columns = ", ".join(filter(None, columns))
-        sql = "SELECT " + selected_columns + " FROM %s" % table_name
-        result = db.engine.connect().execute(sql)
-        return [row for row in result]
-
-    def _get_mapbox_key(self):
-        return conf["MAPBOX_API_KEY"]
+        try:
+            selected_columns = ", ".join(filter(None, columns))
+            sql = "SELECT " + selected_columns + " FROM %s" % table_name
```

Comment:
Vulnerable to SQL-Injection, we can keep this but I'd put a #TODO there so that we don't forget and can look at this if we have the time